workers/loc.api/helpers/get-rest.js

```diff
@@ -141,11 +141,19 @@ module.exports = (conf) => {
     const {
       apiKey = '',
       apiSecret = '',
-      authToken = '',
-      ip = ''
```

Comment:
No ip needed for authToken?

Reply:
yes, it works without `ip`
and looks like `ip` was removed from libs
https://github.com/bitfinexcom/bitfinex-api-node/blob/master/index.js#L22
https://github.com/bitfinexcom/bfx-api-node-rest/blob/master/lib/rest2.js#L85